perfdash/parser.go

```diff
@@ -137,3 +137,58 @@ func parseRequestCountData(data []byte, buildNumber int, testResult *BuildData)
 		}
 	}
 }
+
+type metricValue struct {
+	Metric map[string]string `json: metric`
+	Value  []interface{}     `json: value`
+}
+
+type e2eMeteric []metricValue
+
+type e2eMetrics struct {
+	ApiServerMetrics         map[string]e2eMeteric `ApiServerMetrics`
+	ClusterAutoscalerMetrics map[string]e2eMeteric `ClusterAutoscalerMetrics`
```

Comment:
Can we somehow get rid of this? It looks strange that we're specifically also getting ClusterAutoscalerMetrics.